src/components/MDX/Card.tsx

```diff
@@ -28,7 +29,7 @@ const MDXCard = ({ iiifContent }: { iiifContent: string }) => {
 
   if (!resource) return null;
 
-  return <Card resource={resource} />;
+  return <span><CardStyled><Card resource={resource} /></CardStyled></span>;
```

Comment:
Prefix, and we should be able to wipe away that span.

```suggestion
  return <MDXCardStyled><Card resource={resource} /></MDXCardStyled>;
```